.github/workflows/CI.yml

```diff
@@ -0,0 +1,54 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
You could take a look at Apache Avro Rust SDK CI setup at https://github.com/apache/avro/blob/5b055ac43085b11e467acd66c8630f424b1af50e/.github/workflows/test-lang-rust-ci.yml#L48-L98 for more inspiration.

Also https://github.com/apache/avro/blob/master/.github/workflows/test-lang-rust-clippy.yml and https://github.com/apache/avro/blob/master/.github/workflows/test-lang-rust-audit.yml

Reply:
Thank you so much about your sincerely suggestions ! It just inspire me on how to improve it.